### PR TITLE
5 little typos

### PR DESCRIPTION
**Pull request by kadiwa4**

### CODE REVIEW

File: library/core/src/ptr/const_ptr.rs [hunk 1]
```diff
@@ -132,8 +132,8 @@ impl<T: ?Sized> *const T {
     /// ```
     #[unstable(feature = "ptr_to_from_bits", issue = "91126")]
     #[deprecated(
-        since = "1.67",
-        note = "replaced by the `exposed_addr` method, or update your code \
+        since = "1.67.0",
```
Comment (Nilstrieb) [inline library/core/src/ptr/const_ptr.rs:135]: Hmm, was this not set to `CURRENT_RUSTC_VERSION` and automatically changed on the release 🤔

Comment (kadiwa4) [reply library/core/src/ptr/const_ptr.rs:135]: No (#95583)

Comment (scottmcm) [reply library/core/src/ptr/const_ptr.rs:135]: Oh, oops 🙃 

TBH, these methods should just be deleted now.  The new strict-provenance-related ones are the way forward.

File: tests/codegen/optimize-attr-1.rs [hunk 2]
```diff
@@ -9,7 +9,7 @@
 // CHECK-LABEL: define{{.*}}i32 @nothing
 // CHECK-SAME: [[NOTHING_ATTRS:#[0-9]+]]
 // SIZE-OPT: ret i32 4
-// SPEEC-OPT: ret i32 4
+// SPEED-OPT: ret i32 4
```
Comment (scottmcm) [inline tests/codegen/optimize-attr-1.rs:12]: Oh, nice catch, since this was just a nop with the typo.

**Comment by thomcc** [comment]
Looks fine.

@bors r+ rollup

**Comment by scottmcm** [comment]
Ha, just a couple seconds apart on the `r+`ing

**Pull request merged**